poll_rate: 50       # Rate for all state transition loops, in Hz
call_rate: 30       # Rate for all persistent service calls, in Hz

verbose:            # Verbosity flags for terminal output
  state: true       # If true, state entry and exit messages are published
  transition: true  # If true, transition functions publish info messages

hover_height: 1.5   # Default hovering height, in *metres*
land_height: 2.0    # Height at which the quad lands, in *metres*
distance_error: 0.1 # Min. sq. dist. error to consider a setpoint as achieved, in *metres^2*

agent:              # agent-specific parameters
  x: -5             # x-coordinate of starting position
  y: 1              # y-coordinate of starting position
